arcs/db.py

```diff
@@ -331,7 +331,7 @@ def add_judgments_for_qrps(db_conn, data):
                 raw_judgments.extend(result[0])
 
             cur.execute(update, (row["judgment"], row["_golden"],
-                                 raw_judgments, query, result_fxf))
+                                 simplejson.dumps(raw_judgments), query, result_fxf))
```

Comment:
Why are there three things in `update` (judgment, is_gold, raw_judgments), and five things that get passed in (judgment, golden, raw_judgment, query, result fxf)?